### PR TITLE
style(badge.scss): added/modify style for badge component

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
# Proposed Changes

+border-radius, +text-transform: uppercase, *font-size, *height

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by GermainBergeron** [comment]
I think the text is not centered which kind of look bad

image

**Comment by GermainBergeron** [comment]
The demo is not updated, can you provide a screenshot of the navigation & a screenshot of the badge?

File: packages/vapor/scss/components/badge.scss [hunk 1]
```diff
@@ -6,10 +6,15 @@
...
-    line-height: $badge-height;
+    line-height: $badge-line-height;
+    text-transform: uppercase;
     background-color: $orange;
 
     &:last-child {
         margin-right: 0;
     }
+
+    &.mod-small {
```
Comment (gdostie) [inline packages/vapor/scss/components/badge.scss]: add an example for mod-small ?

**Comment by FelixBlaisThon** [comment]
In the nav 
image

In Badge component
image

**Pull request merged**